Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -155,7 +155,7 @@ bool SVGAnimationElement::attributeContainsJavaScriptURL(const Attribute& attrib
 
     if (attribute.name() == SVGNames::valuesAttr) {
         for (auto innerValue : StringView(attribute.value()).split(';')) {
-            if (WTF::protocolIsJavaScript(innerValue.stripWhiteSpace()))
+            if (WTF::protocolIsJavaScript(stripLeadingAndTrailingHTMLSpaces(innerValue.toString())))
```

Comment:
My fault, I pointed you at this function, but we really don’t want to use the one that requires converting the `StringView` to a `String`, then the result to another `String`, then back to a `StringView`. So instead we want to use `innerValue.stripLeadingAndTrailingMatchedCharacters(isHTMLSpace<UChar>)`. Sorry for steering you wrong.

Also, I wish the two nearly identical functions weren’t called in such different ways.